OMPlot/OMPlotGUI/PlotWindow.cpp

```diff
@@ -41,9 +41,15 @@
 
 using namespace OMPlot;
 
-PlotWindow::PlotWindow(QStringList arguments, QWidget *parent)
-  : QMainWindow(parent)
+PlotWindow::PlotWindow(QStringList arguments, QWidget *parent, bool isInteractiveSimulation, QToolButton *pStartSimulation,
+                       QToolButton *pPauseSimulation, QComboBox *pSimulationSpeed)
+  : QMainWindow(parent), mIsInteractiveSimulation(isInteractiveSimulation)
```

Comment:
This is weird. Why are you creating the buttons in OMEdit and passing them here. Just `bool isInteractiveSimulation` is enough and then create buttons based on that. Access the buttons in OMEdit via getters and then connect them to some slot.